src/drivers/bst/bst.cpp

```diff
@@ -289,11 +290,13 @@ void BST::cycle()
 		if (updated) {
 			vehicle_attitude_s att;
 			orb_copy(ORB_ID(vehicle_attitude), _attitude_sub, &att);
+			matrix::Quaternion<float> q(&att.q[0]);
```

Comment:
I think that &att.q[0] and att.q are equivalent, both a pointer to the first entry, so might look better to type:

matrix::Quaternion<float> q(att.q);

Also, maybe more readable to use

using namespace matrix;

Quatf q(att.q);